awx/api/serializers.py

```diff
@@ -3600,7 +3600,7 @@ def validate(self, attrs):
             ujt = self.instance.unified_job_template
         if ujt is None:
             ret = {}
-            for fd in ('workflow_job_template', 'identifier'):
+            for fd in ('workflow_job_template', 'identifier', 'all_parents_must_converge'):
```

Comment:
Could we add a functional/API test for this?

Reply:
Maybe under `awx/main/tests/functional/api/test_workflow_node.py` ?

Reply:
added a functional test